src/pages/en/guides/markdown-content.md

```diff
@@ -178,32 +177,72 @@ You can import components into your Markdown file with `setup` and use them alon
 ```markdown
 ---
 layout: ../layouts/BaseLayout.astro
-setup: | 
+setup: |
   import Author from '../../components/Author.astro'
   import Biography from '../components/Biography.jsx'
 author: Leon
 ---
+
 <Author name={frontmatter.author}/>
 <Biography client:visible>
   {frontmatter.author} lives in Toronto, Canada and enjoys photography.
 </Biography>
 ```
 
-<!-- 
-
 ## Importing Markdown
 
-https://github.com/withastro/rfcs/pull/147
+You can import Markdown files directly into your Astro files! You can import one specific page with `import` or multiple with `Astro.glob()`
 
-TODO: This RFC will change how we import markdown, if accepted.
-Therefore, I'd recommend not writing this section now and coming
-back to it later.
+```astro
+---
+// You can use `import ... from`
+import * as greatPost from '../pages/post/great-post.md';
+
+// Or even `await import()`
+const aboutUs = await import('../pages/internal/about-us.md');
+
+// Lastly, you can import multiple files with Astro.glob
+const posts = await Astro.glob('../pages/post/*.md');
+---
+
+Great post: <a href={greatPost.url}>{greatPost.frontmatter.title}</a>
+
+<h2>About us</h2>
+<aboutUs.Content />
+
+<ul>
+  {posts.map(post => <li>{post.frontmatter.title}</li>)}
+</ul>
+```
 
--->
+Each one of the examples above return an (or multiple) `MarkdownInstance`, an object with the following properties:
```

Comment:
```suggestion
Each one of the examples above return a (or multiple) `MarkdownInstance`, an object with the following properties:
```

Reply:
I *think* this would be correct, even though it might seem to go against the prononciation rule.

Another way around this could be something ilke, "Each Markdown file imported returns a `MarkdownInstance`..."